cipher/caesar_test.go

```diff
@@ -1,11 +1,13 @@
-package caesar
+package cipher_test
```

Comment:
Maybe add the new fuzzing tests for the ciphers, by encrypting and then decrypting and checking equality with the cleartext?

Reply:
I am in favour of using `FuzzXxx` but from what I have seen - they are a bit slow, plus I won't have time to do any significant improvement for a while other than small refactoring 😓.

Maybe we should open a separate issue so that contributors can add Fuzz tests.

Reply:
Sure. Though I also can't work on them right now with exams going on. But they will be useful in the future.

Reply:
Yeah, don't worry - it would be open to all contributors not just maintainers with a `help-wanted` tag and hopefully if someone sees it then they can contribute if not, we – maintainers – can add them whenever we get some time.